boruta/readme.md

```diff
@@ -0,0 +1,30 @@
+# Boruta Feature Selection
+
+In this package you'll find two scripts implementing Boruta Feature Selection:
+
+- [Boruta 1-click Feature Selection](./1-click-boruta)
+  Starting from a dataset, creates a new one removing the unimportant features
+  according to the Boruta Feature Selection algorithm.
+- [Boruta Feature Selection](./boruta)  General Boruta Feature Selection
+  algorithm to rank fields from a dataset into three categories: important,
+  unimportant and undecided.
```

Comment:
Well, the output of the original boruta script is just a map of "important", "unimportant" and "undecided" fields. The 1-click creates a new dataset removing the fields marked as unimportant and gives defaults for every argument except the datase ID, so objective field is the one in the original dataset, the maximum number of runs is 10 and the epsilon arount the maximum importance of the shadow fields is 0.01